spec.md

```diff
@@ -721,30 +721,6 @@ f(-1)           # returns 1 without printing
 ```
 
 
-It is a dynamic error for a function to call itself or another
```

Comment:
The description of the check belongs here, under function calls, even if the feature is optional. 

It's fine to add an optional features section at the end, but it should be a summary of places in the doc where optional features are described.

Reply:
Moved back.